content/shmem_broadcast_nb.tex

```diff
@@ -0,0 +1,113 @@
+\apisummary{
+    Broadcasts a block of data from one \ac{PE} to one or more destination
+    \acp{PE}.
+}
+
+\begin{apidefinition}
+
+%% C11
+\begin{C11synopsis}
+int @\FuncDecl{shmem\_broadcast\_nb}@(shmem_team_t team, TYPE *dest, const TYPE
+*source, size_t nelems, int PE_root, uint32_t tag, shmem_req_h *request);
+\end{C11synopsis}
+where \TYPE{} is one of the standard \ac{RMA} types specified by Table \ref{stdrmatypes}.
+
+%% C/C++
+\begin{Csynopsis}
+\end{Csynopsis}
+\begin{CsynopsisCol}
+int @\FuncDecl{shmem\_\FuncParam{TYPENAME}\_broadcast\_nb}@(shmem_team_t team, TYPE
+*dest, const TYPE *source, size_t nelems, int PE_root, uint32_t tag,
+shmem_req_h *request);
+\end{CsynopsisCol}
+where \TYPE{} is one of the standard \ac{RMA} types and has a corresponding \TYPENAME{} specified by Table \ref{stdrmatypes}.
+
+\begin{CsynopsisCol}
+int @\FuncDecl{shmem\_broadcastmem\_nb}@(shmem_team_t team, void *dest, const void
+*source, size_t nelems, int PE_root, uint32_t tag, shmem_req_h *request);
+\end{CsynopsisCol}
+
+\begin{apiarguments}
+
+\apiargument{IN}{team}{The team over which to perform the operation.}%
+
+\apiargument{OUT}{dest}{Symmetric address of destination data object.
+      The type of \dest{} should match that implied in the SYNOPSIS section.}
+\apiargument{IN}{source}{Symmetric address of the source data object.
+      The type of \source{} should match that implied in the SYNOPSIS section.}
+\apiargument{IN}{nelems}{
+  The number of elements in \source{} and \dest{} arrays.
+  For \FUNC{shmem\_broadcastmem\_nb}, elements are bytes;
+  for \FUNC{shmem\_broadcast\{32,64\}\_nb}, elements are 4 or 8 bytes,
+  respectively.
+}
+\apiargument{IN}{PE\_root}{Zero-based ordinal of the \ac{PE}, with respect to
+    the team, from which the data is copied.}
+\apiargument{IN}{tag}{A user defined tag to order the collective operation;
+SHMEM\_COLL\_UNORDERED can be provided if no order is required.}
+\apiargument{OUT}{request}{An opaque request handle identifying the collective
+operation.}
+
+
+\end{apiarguments}
+
+\apidescription{   
+    \openshmem nonblocking broadcast routines are collective routines over a 
+    valid \openshmem team.
+    They copy the \source{} data object on the \ac{PE} specified by
+    \VAR{PE\_root} to the \dest{} data object on the \acp{PE}
+    participating in the collective operation.
+    The same \dest{} and \source{} data objects and the same value of
+    \VAR{PE\_root} must be passed by all \acp{PE} participating in the
+    collective operation.
+
+    A call to the nonblocking broadcast routine posts the operation and returns
+    immediately without necessarily completing the operation. On the successful
+    post of the operation, an opaque request handle is created and returned. The
+    operation is completed after a call to \FUNC{shmem\_req\_test} or
+    \FUNC{shmem\_req\_wait}. When the operation is complete, the request handle
+    is deallocated and cannot be reused.
```

Comment:
To my understanding. This is currently worded in a way that both `shmem_req_test` and `shmem_req_wait` guarantee completion.

I believe the correct language exists in the intro on lines 17 and 18.